CircleCI Commandset - feedback changes implemented (#218)
nimbella/command-sets: circleci/README.md

# Circle CI

### Install

> Run the command below to install the whole CircleCI command set. 

Reviewer: ```suggestion
Run the command below to install the whole CircleCI command set.
```

```
/nc csm_install circleci
```

### Prerequisite:
You will need to create the following 3 secrets to use this command set.

> Those are :

Reviewer: ```suggestion
```

> `projectToken` - This is your CircleCI API access token, Get yours here https://circleci.com/docs/2.0/managing-api-tokens/#creating-a-personal-api-token

Reviewer: ```suggestion
- `projectToken`: This is your CircleCI API access token. You may generate one here https://circleci.com/docs/2.0/managing-api-tokens/#creating-a-personal-api-token.
```

> This secret should be created with the following naming convention. `projectName_token`. Where the project name is your circleCi project name. example `webApp_token`.

Reviewer: ```suggestion
This secret should be created with the following naming convention. `projectName_token`. Where the project name is your CircleCi project name. example `webApp_token`.
```

> `vcsType` - Version Control System i.e. either `gh` for GitHub or `bb` BitBucket. This is optional and will be assumed as `gh` by default.

Reviewer: ```suggestion
- `vcsType`: The Version Control System. Only two values are currently supported. They are `gh` for GitHub, and `bb` for BitBucket. This property is optional and defaults to GitHub.
```

> `orgName` - Name of your GitHub or BitBucket organization or else account username if its a personal project. Creating a secret is optional and can be passed in command parameters, But, passing it in the parameter has a higher priority than a secret.

Reviewer: ```suggestion
- `orgName`: The name of your GitHub or BitBucket organization, or else your account username for personal projects. Creating this secret property is optional but convenient when you are working with just one organization. If you are using the command set with multiple organizations, you may want to pass this value as a command parameter. Note that using a command parameter for this property supersedes the secret.
```
